add-chain/cmd/update-configs.go

```diff
@@ -0,0 +1,40 @@
+package cmd
+
+import (
+	"path/filepath"
+	"runtime"
+
+	"github.com/ethereum-optimism/superchain-registry/add-chain/config"
+	"github.com/ethereum-optimism/superchain-registry/superchain"
+	"github.com/urfave/cli/v2"
+)
+
+var UpdateConfigsCmd = cli.Command{
```

Comment:
We can delete this whole file if we want. But could be useful to automate tasks such as adding hardfork timestamps in the future

Reply:
Leaving this in for future use